pages/common/identify.md

```diff
@@ -3,6 +3,6 @@
 > Command-line utility of Image Magick project to describe the format and characteristics of one or more image files.
 > More information: <https://imagemagick.org/script/identify.php>.
 
-- Collect dimensions of all jpeg files under current directory:
+- Collect dimensions of all JPEG files under current directory:
```

Comment:
This could work too, no?
```suggestion
- Collect dimensions of all `.jpg` files under current directory:
```

Reply:
That's valid, is that input specific to this command, or for other instances or file types as well?

Reply:
No, it can use any ImageMagick-supported file type — there are many.  The example is using `.jpg`.

Reply:
Oh nono, I'm familiar with ImageMagick.
I meant is your feedback for this command only, or for every instance of `PNG`, `JPG`, `PDF`, `PSD`, etc I changed.

As in do you think they should all be `.jpg`, `.png`, `.whatever`?
Or is the feedback just for this command since the command example uses `.jpg`?

Reply:
Ah, I see.  I guess it's for all commands but that seems like a lot of work — only do it for all them if you really want to.